odl/solvers/nonsmooth/douglas_rachford.py

```diff
@@ -28,7 +28,7 @@
 __all__ = ('douglas_rachford_pd',)
 
 
-def douglas_rachford_pd(x, prox_f, prox_cc_g, L, tau, sigma, niter,
+def douglas_rachford_pd(x, f, g, L, tau, sigma, niter,
```

Comment:
I agree with @adler-j. I don't think we can name the functionals anything that will make it more clear what they are, if you do not read the documentation. And this naming is (at least, if I recall correctly) consistent with the original paper which we cite further down.